tests/pages/catalog/packages/package/PackageTab-cy.js

```diff
@@ -226,7 +226,7 @@ describe("Package Detail Tab", function() {
             .should("not.exist");
         });
 
-        it('Should hide the "value" when "Unique" is selected in operator dropdown', function() {
+        it("Should disable the field value when Unique is selected in operator dropdown", function() {
```

Comment:
Not important, since we've edited this, we could remove the should.

Reply:
Agree, I just didn't remove because I would have to remove for all of the tests and didn't want to create more noise in the PR